apache-rat-core/src/main/resources/org/apache/rat/default.xml

```diff
@@ -158,7 +158,7 @@
 					may
 					be prepared copied published and distributed</text>
 				<copyright
-					owner="OASIS Open" start="[0-9]{4}" />
+					owner="OASIS Open"/>
```

Comment:
What is the semantics of this change? Should there be a test for this OASIS license that needs to be changed here?